src/grammar.json

```diff
@@ -1505,5 +1633,6 @@
     ],
     "num0-9": ["0","1","2","3","4","5","6","7","8","9"],
     "num1-9": ["1","2","3","4","5","6","7","8","9"],
-    "num2-9": ["2","3","4","5","6","7","8","9"]
+    "num2-9": ["2","3","4","5","6","7","8","9"],
+    "colon": ":"
```

Comment:
tracery doesn't correctly parse inline rules with strings that contain colons :poultry_leg: